### PR TITLE
[INLONG-7174][Agent] Support converting DataConfig to TriggerProfile for PostgreSQL

### LINKED ISSUE #7174: [Feature][Agent] Support converting DataConfig to TriggerProfile for PostgreSQL
### Description

Support converting DataConfig to TriggerProfile for [PostgreSQL](https://www.postgresql.org/)

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
### Prepare a Pull Request

- Fixes #7174

### Motivation

Support converting DataConfig to TriggerProfile for PostgreSQL.

### Modifications

Add `PostgresJob` and `POSTGRES` case.

### CODE REVIEW

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgresJob.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        private String hostname;
+        private Integer port;
+        private String database;
+        private String schema;
+        private String decodingPluginName;
+        private List<String> tableNameList;
+        private String serverTimeZone;
+        private String scanStartupMode;
+        private String primaryKey;
+        private Map<String, Object> properties;
```
Comment (pocozh) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgresJob.java:58]: Please make sure the types of `port`, `tableNameList` and `properties` from InLong-Manager.

Comment (haifxu) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgresJob.java]: Type conversion has been added.

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgresJob.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * PostgresJob : postgres job
+ */
+@Data
+public class PostgresJob {
```
Comment (pocozh) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgresJob.java:29]: Class name and field names should be consistent with definitions in `PostgreSQLReader`

Comment (haifxu) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgresJob.java]: Done.

**Review by pocozh — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by woofyzhao — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/JobProfileDto.java [hunk 3]
```diff
@@ -52,6 +52,10 @@ public class JobProfileDto {
      * kafka source
      */
     public static final String KAFKA_SOURCE = "org.apache.inlong.agent.plugin.sources.KafkaSource";
+    /**
+     * Postgres source
+     */
+    public static final String POSTGRES_SOURCE = "org.apache.inlong.agent.plugin.sources.PostgreSQLSource";
```
Comment (healchow) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/JobProfileDto.java:58]: Postgres -> PostgreSQL

POSTGRES_SOURCE -> POSTGRESQL_SOURCE

Comment (haifxu) [reply inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/JobProfileDto.java]: Done.

File: inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgreSQLJob.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ */
+
+package org.apache.inlong.agent.pojo;
+
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * PostgresJob : postgres job
```
Comment (healchow) [inline inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/pojo/PostgreSQLJob.java:25]:
```suggestion
 * PostgreSQL Job info
```

**Review by healchow — APPROVED** [review]

**Pull request merged**